### PR TITLE
Feature/scope change v3

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Components/DetailView/RequestDetailView.tsx [hunk 1]
```diff
@@ -163,6 +163,33 @@ export const RequestDetailView = ({
...
+                        <div>
+                            {request.attachments &&
+                                request.attachments.map((x) => {
+                                    return (
+                                        <a
+                                            href={`https://app-ppo-scope-change-control-api-dev.azurewebsites.net/api/scope-change-requests/${request.id}/attachments/${x.id}`}
+                                            download
+                                            style={{
+                                                display: 'flex',
+                                                color: `${tokens.colors.interactive.primary__resting.rgba}`,
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/DetailView/RequestDetailView.tsx:179]: styled component

File: src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx [hunk 2]
```diff
@@ -140,8 +156,31 @@ export const ScopeChangeRequestForm = ({
...
+                        );
+                    })}
+
+                {/* <Field
+                    label="Attachments"
+                    value={<Upload attachments={attachments} setAttachments={setAttachments} />}
+                /> */}
             </GeneratedForm>
             {error && <p> Something went wrong, please check your connection and try again</p>}
         </FormContainer>
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Form/ScopeChangeRequestForm.tsx:186]: fragment

File: src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx [hunk 3]
```diff
@@ -1,92 +1,151 @@
...
+                            theme={applyEDSTheme}
+                        />
+                        {documents &&
+                            documents.map((x) => {
+                                return (
+                                    <Chip key={x.value}>
+                                        <StidDocument document={x.object as Document} />
+
+                                        <Icon
+                                            color={tokens.colors.interactive.primary__resting.rgba}
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchableDropdown/stidSelector.tsx:101]: wrap in button or make pointer

File: src/apps/ScopeChangeRequest/Components/Upload.tsx [hunk 4]
```diff
@@ -41,20 +54,47 @@ export const Upload = (): JSX.Element => {
...
+                            />
+                        </Inline>
+                    </AttachmentsList>
+                );
+            })}
+            {rejectedFiles && rejectedFiles.length > 0 && (
+                <div style={{ color: 'red' }}>
+                    The following files could not be added
+                    {rejectedFiles.map((x) => {
+                        console.log(x);
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/Upload.tsx:89]: Remove console.logs

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**